docs/docker/run.md

```diff
@@ -1,41 +1,39 @@
-## Run you own __Cardano Node__ 
-
-## OS Requirements
+### OS Requirements
 
 - "docker-ce" installed.
 
-## Private mode
+### Private mode
 
-### use cases
+#### Use Cases
 
 - Pool Management
 - Wallet Management
 - Node testing
 
 ```bash
-docker run -dit 
--e NETWORK=mainnet 
---name <YourCName>
--p <your_custom_path>:/opt/cardano/cnode/priv
--p <your_custom_db_path>:/opt/cardano/cnode/db
+docker run -dit \
+-e NETWORK=mainnet \
+--name <YourCName> \
+-p <your_custom_path>:/opt/cardano/cnode/priv \
+-p <your_custom_db_path>:/opt/cardano/cnode/db \
 cardanocommunity/cardano-node 
 ```
 
-## Public mode
+### Public mode
 
-### use cases:
+#### Use Cases:
 
 - Node Relay
 
 ```bash
-docker run -dit 
---name <YourCName> 
--e NETWORK=mainnet
--p 6000:6000
--e NETWORK=mainnet  
--p <your_custom_path>:/opt/cardano/cnode/priv
--p <your_custom_db_path>:/opt/cardano/cnode/db
-cardanocommunity/cardano-node 
+docker run -dit \ 
+--name <YourCName> \
+-e NETWORK=mainnet \
+-p 6000:6000 \
+-e NETWORK=mainnet \
+-p <your_custom_path>:/opt/cardano/cnode/priv \
```

Comment:
Same here, above it properly uses `-p 6000:6000` on line 32, but then re-uses -p instead of -v on line 34 and 35.

Reply:
Resolved on base PR